ui/oopsyraidsy/oopsy_common.ts

```diff
@@ -169,7 +169,7 @@ export const GetSoloMistakeText = (ability: string): LocaleText => {
 export const GetShareMistakeText = (ability: string, numTargets: number): LocaleText => {
   return {
     en: `${ability} (share x${numTargets})`,
-    de: `${ability} (geteilt)`, // FIXME
+    de: `${ability} (geteilt)`,
```

Comment:
Sorry, I didn't know how other languages wanted to include `numTargets` here.  Can you add that?